src/components/InputField/InputField.tsx

```diff
@@ -292,12 +297,24 @@ export const InputField: InputFieldType = forwardRef(
             ref={ref}
             required={required}
             status={shouldRenderError ? 'critical' : status}
-            type={type}
+            type={isPasswordVisible ? 'text' : type}
```

Comment:
Not having the type="password" may mess with the usage of `autocomplete`, but since we're using Google Login for Render I don't think this is an issue

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password#allowing_autocomplete